simulator/cube.py

```diff
@@ -294,12 +294,24 @@ def snap_to_grid(self, grid_size, others, offset = 0):
         grid_size: pixel size of grid as (w, h)
         others: List of other cubes the current cube can snap to
         offsest: offset of the snap grid in pixels (x, y) """
-
+	
```

Comment:
Remove trailing whitespace. You might have to turn on highlighting this in your IDE.